libs/s3/s3_misc.py

```diff
@@ -163,32 +163,6 @@ def delete_objects_bucket(bucket_name, access_key: str, secret_key: str, **kwarg
     LOGGER.debug("Verified bucket is deleted.")
     return not result
 
-def delete_object(bucket_name, obj_name, access_key: str, secret_key: str, **kwargs):
```

Comment:
is this not getting used by anyone?

Reply:
No. There is a similar function which is in use by some of my tests from previous PR.